### PR TITLE
removed Lifecylelistener from mancenter service and fixed ManagementC…

### PR DESCRIPTION
**Pull request by hasancelik**
…enterServiceIntegrationTest.testTimedMemberStateNotNull

### CODE REVIEW

**Comment by hasancelik** [comment]
run-lab-run

File: hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java [hunk 1]
```diff
@@ -164,7 +157,7 @@ static String cleanupUrl(String url) {
         return url.endsWith("/") ? url : url + '/';
     }
 
-    public void start() {
+    private void start() {
```
Comment (mdogan) [inline hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java]: I think you should check whether management center is enabled.

```java
        if (!managementCenterConfig.isEnabled()) {
            return;
        }
```

Comment (mdogan) [inline hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java]: Also you shouldn't register membership listener if not enabled.

Comment (hasancelik) [inline hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java]: Yes, you are right. fixed https://github.com/hazelcast/hazelcast/pull/9571/commits/f75fc2f44a0778c910e3f1f9e0f404368446cde3#diff-3850ee7b4bd9df3672e6c4b18d1ec70cR131

**Pull request merged**